apps/next/src/app/components/nav/nav.ts

```diff
@@ -27,31 +32,17 @@ import { ThemeService } from '../../theme.service';
   },
 })
 export class Nav {
+  _navData$ = this._pageService._getCategories().subscribe((data) => {
+    this._navItems.push(...data);
+    this._changeDetectorRef.markForCheck(); // Angular doesn't understand that the array values changed.
+  });
```

Comment:
Please either subscribe in the constructor or in ngOnInit.
Also don't forget to unsubscribe.